### DISCUSSION TITLE
Possible to get beta release with latest MQTT patch?

### DISCUSSION THREAD

**Discussion by bartowski1182** [alandtse/tesla#567] [General]
Really want to try the new linking to teslamate MQTT, how long is the lifecycle between commit and release? Can we get a beta?

**Comment by alandtse** [comment]
The beta is live. Just pull dev in HACS.

**Reply by bartowski1182** [reply]
forgive my ignorance, I don't see a way to change which release my integration in HACS is (and searching has failed me), do I have to uninstall first?

**Reply by alandtse** [reply]
https://hacs.xyz/docs/faq/select_version/
